src/main/java/com/github/kongchen/swagger/docgen/mavenplugin/SpringMavenDocumentSource.java

```diff
@@ -45,6 +46,9 @@ public void loadDocuments() throws GenerateException {
         }
 
         swagger = resolveApiReader().read(apiSource.getValidClasses(Api.class));
+        if (swagger.getPaths() == null) {
+          swagger = resolveApiReader().read(apiSource.getValidClasses(RestController.class));
```

Comment:
Wouldn't it be better to search all Api and all RestController classes instead of one or the other? What if I have 5 classes annotated @RestController and one has also the @Api annotation?

I would suggest changing it to:
```
Sets.SetView<Class<?>> validClasses = Sets.union(
            apiSource.getValidClasses(Api.class), 
            apiSource.getValidClasses(RestController.class));
swagger = resolveApiReader().read(validClasses);
```